stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -3435,12 +3511,13 @@ final internal class _SwiftDeferredNS${Self}<${TypeParametersDecl}>
     super.init()
   }
 
-  // This stored property should be stored at offset zero.  We perform atomic
-  // operations on it.
-  //
-  // Do not access this property directly.
+  /// This stored property should be stored at offset zero.  We perform atomic
```

Comment:
There isn't much to say about it on its own; I'll just change it to a "normal" comment.